lib/Conversion/TorchToTcp/TcpCustomOp.cpp

```diff
@@ -146,6 +147,39 @@ class ConvertAten_IndexPutImplOp
   }
 };
 
+class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
+public:
+  using OpConversionPattern<AtenConvolutionOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(AtenConvolutionOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Type> resultTypes;
+    if (failed(OpConversionPattern<AtenConvolutionOp>::getTypeConverter()
+                   ->convertTypes(op->getResultTypes(), resultTypes))) {
+      return failure();
+    }
+
+    SmallVector<Value> operands;
+    operands.push_back(adaptor.getInput());
```

Comment:
Actually, I wonder if there's just a generic iterator over all the input parameters of the original op so I do not have to write this down manually. Will get back to this in a bit.